Assets/MixedRealityToolkit/MSBuild/MSBuildTemplates/SDKProjectTemplate.csproj

```diff
@@ -0,0 +1,119 @@
+<Project>
+  <PropertyGroup>
+    <!--anborod: This is a weird thing, it is a required property (even if commented)-->
+    <!--<TargetFrameworks>netstandard2.0;uap10.0;net46</TargetFrameworks> -->
```

Comment:
>!--<TargetFrameworks>netstandard2.0;uap10.0;net46</TargetFrameworks> --> [](start = 5, length = 72)

If this is omitted, is it pretty obvious about the failures that start happening, or it is pretty obscure? This comment isn't asking for any changes here I'm just curious to know the story behind it.

Reply:
Yeah, project won't load, it's strange.

---
In reply to: [301778093](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5215#discussion_r301778093) [](ancestors = 301778093)